File: 0001-two-sum/0001-two-sum.java
class Solution {
    static int[] twoSum(int[] nums, int target) {
        int i=0;
        int j=0;
        for (i = 0; i <= nums.length-1; i++) {
            for ( j = 0; j <=nums.length-1 ; j++) {
                if(nums[i]+nums[j]==target) {
                    if(i<j)
                    {
                        return new int[]{i,j};
                    }
                
                }
                }
            }
        return null;
    }
}

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    const ans =[];
    for(let i=0;i<nums.length;i++){
        for(let j=0;j<nums.length;j++){
            if(i!=j && nums[i]+nums[j]==target)
            {
                ans[0]=i;
                ans[1]=j;
                return ans;
            }
        }
    }
    return ans;
};